airlock/file_browser_api.py

```diff
@@ -233,15 +272,21 @@ def get_workspace_tree(workspace, selected_path=ROOT_PATH, selected_only=False):
 
     if selected_only:
         pathlist = [selected_path]
+        directories = set()
 
-        # if directory, we also need to also load children to display in the content area
+        # if directory, we also need to also load our children to display in
+        # the content area. We don't mind using stat() on the children here, as
+        # we are only loading a single directory, not an entire tree
         abspath = workspace.abspath(selected_path)
         if abspath.is_dir():
-            pathlist.extend(child.relative_to(root) for child in abspath.iterdir())
+            for child in abspath.iterdir():
+                path = child.relative_to(root)
+                pathlist.append(path)
+                if child.is_dir():
+                    directories.add(path)
```

Comment:
here `directories` is all dir paths, whether empty or not

Reply:
good point, I will fix to be just empty, for consistency.

This part of the tree is *only* used to partially render a directory contents - not render the actually tree, so in practice, I'm not sure it makes a difference.

But I think it makes sense to be consistent :+1: